docs/documentation.md

```diff
@@ -265,6 +265,39 @@ This example shows you how to create a very simple search results page that incl
 
 <img class="img-responsive" src="../img/add-widgets.jpg" alt="Add widgets">
 
+### Events
+
+<div class="code-box">
+  <div class="code-sample-snippet js-toggle-snippet ignore">
+{% highlight js %}
+var search = instantsearch({
+  appId: '$appId',
+  apiKey: '$apiKey',
+  indexName: '$indexName',
+  urlSync: true
+});
+
+var onRenderHandler = function() {};
+search.on('render', onRenderHandler);
+// on renderHandler will be called
+// until removeListener is called
+search.removeListener(onRenderHandler);
+
+search.once('render', function(){  });
+// triggered once then removed automatically
+{% endhighlight %}
+  </div>
+</div>
+
+InstantSearch emits event during its lifecylce. The currently supported events are:
+
+ - `render` : fired when a rendering of all the widgets has been completed
+
+InstantSearch events are based on the [Node EventEmitter](https://nodejs.org/api/events.html)
+class. See the example on the right for a quick overview of the API or go to the
```

Comment:
Careful because "see the example on the right" only works at certain screen size